packages/pharos/src/components/pagination/pharos-pagination.scss

```diff
@@ -8,12 +8,27 @@
 .pagination__wrapper {
   width: 100%;
   display: inline-flex;
+  gap: var(--pharos-spacing-1-x);
+  align-items: center;
 }
 
 .pagination__info {
   padding: 0 var(--pharos-spacing-1-x);
 }
 
+.pagination__input-wrapper {
+  display: inline-flex;
+  align-items: center;
+}
+
+.pagination__input {
+  height: 34px;
```

Comment:
We usually try to avoid setting exact pixel sizes, I'm curious is this really _needs_ to be shorter/taller than our normal input?